Analysis/Tutorials/src/schemaEvolution.cxx

```diff
@@ -26,15 +26,15 @@ DECLARE_SOA_COLUMN(Eta, eta, float);
 DECLARE_SOA_COLUMN(AbsEta, absEta, float);
 DECLARE_SOA_COLUMN(Phi, phi, float);
 } // namespace etaphi
-DECLARE_SOA_TABLE(EtaPhiV1, "AOD", "ETAPHI", etaphi::Eta, etaphi::Phi);
+DECLARE_SOA_TABLE(EtaPhiV1, "AOD", "ETAPHIV1", etaphi::Eta, etaphi::Phi);
```

Comment:
In principle this was on purpose because when you have made ETAPHI (v1) you did not yet know that there will be a version 2. However, if you find it more clear this way, I can also live with it.

Reply:
Let's keep with V1 / V2